manifests/config.pp

```diff
@@ -34,6 +34,12 @@
         }
       }
       'systemd' : {
+        file { '/etc/sysconfig/nomad':
+          ensure  => directory,
```

Comment:
Can you fix the indentation on line 38-41? Puppetlint will complain about it otherwise.